modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -90,7 +90,7 @@ controlPlane:
 
 ifeval::[{product-version} >= 4.4]
 a| [[provisioningNetworkInterface]]`provisioningNetworkInterface` |  | The name of the network interface on control plane nodes connected to the
-provisioning network.
+provisioning network. Optional as of {product-title} 4.4 and later releases, due the `bootMacAddress` field calculating the IP address for Ironic to bind to.
```

Comment:
I wonder would it be better to restructure this sentences, like the other instances below.
Maybe:
As of {product-title} 4.4 and later releases, this filed is now optional due to the `bootMacAddress` field calculating the IP address for Ironic to bind to.

Reply:
Excellent suggestion! I've updated that line.